elementary/monitor/alerts/test.py

```diff
@@ -318,6 +332,17 @@ def to_slack(self, is_slack_workflow: bool = False) -> SlackMessageSchema:
                 ),
             )
 
+        test_runs_report_link = get_test_runs_report_link(
```

Comment:
maybe i'd rename that too just so that it's clear that it's meant for the alerts and not the report. something like `get_test_runs_url_for_alert`

Reply:
This file is under `monitor/alerts`, do you think `for_alert` should still be added to the name?